make.py

```diff
@@ -719,12 +720,13 @@ def _create_batch_scripts_initial(self):
         self.create_batch_script('cmd_ps.bat', r"""@echo off
 rem safe bet 
 call "%~dp0env_for_icons.bat"
-Powershell.exe -executionpolicy RemoteSigned -noexit -file "%~dp0WinPython_PS_Prompt.ps1"
+Powershell.exe -Command "& {Start-Process PowerShell.exe -ArgumentList '-ExecutionPolicy RemoteSigned -noexit -File""%~dp0WinPython_PS_Prompt.ps1""'}"
 """)

```

Comment:
II think `exit` must be inserted on this line (line above 724) in order to exit the cmd.exe (the powershell console remains open), see line 729.